src/lib/wallet/GoodWallet.js

```diff
@@ -40,6 +32,13 @@ type QueryEvent = {
 }
 
 export class GoodWallet {
+  static AccountUsageToPath = {
+    gd: 1,
+    gundb: 2,
+    eth: 3,
+    donate: 4,
+    login: 4
+  }
```

Comment:
Why is `gd` no longer address `0`? 

is it intentional that `donate` and `login` both use the same address?

Reply:
restored. I already start at address 1 when generating the addresses from hdwallet. the first public address of hdwallet can be used to derive all the other public addresses